network-api/networkapi/wagtailpages/models.py

```diff
@@ -486,6 +487,43 @@ def get_context(self, request):
         return get_page_tree_information(self, context)
 
 
+class BanneredCampaignPage(PrimaryPage):
+    """
+    title, header, intro, and body are inherited from PrimaryPage
+    """
+
+    # Note that this is a different related_name, as the `page`
+    # name is already taken as back-referenced to CampaignPage.
+    cta = models.ForeignKey(
+        'Petition',
+        related_name='bcpage',
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        help_text='Choose existing or create new sign-up form'
+    )
+
+    content_panels = PrimaryPage.content_panels + [
```

Comment:
does it make sense to put this in between the parent panels, so that the cta is above the body field?

Reply:
Just moving it up one notch? 
<img width="1023" alt="image" src="https://user-images.githubusercontent.com/1682681/53530092-cd0db100-3abc-11e9-8b64-63c68a83d479.png">

I don't think that's necessary, but a designer might have a thought.

Reply:
yeah, since the body can run pretty long. Then again, if we don't want CTAs on every single of these pages, it might make sense to keep it at the bottom instead?